currencycom/client.py

```diff
@@ -77,22 +41,30 @@ class TimeInForce(Enum):
     GTC = 'GTC'
 
 
+class ExpireTimestamp(Enum):
+    DEFAULT = 0
+    GTC = 'GTC'
+    FOK = 'FOK'
+
+
 class NewOrderResponseType(Enum):
     ACK = 'ACK'
     RESULT = 'RESULT'
     FULL = 'FULL'
 
 
-class Client(object):
+class CurrencycomClient:
```

Comment:
To avoid breaking anyone who works with this API I would add `Client` inherited by `CurrencycomClient` with deprication warning